### ISSUE TITLE
large submission size failed at file_upload

### ISSUE THREAD

**Issue by jimentwood** [arXiv/arxiv-submission-ui#100]
I tried a 56M tar file and got an error
413 Request Entity Too Large
nginx/1.15.2

Maybe just an issue with testing on development.arxiv.org... ?

**Comment by erickpeirson** [comment]
Yeah, limit is currently set to 50MB. Given that this is way outside the allowed ranges, should we increase the limit, or provide an informative error message?

**Comment by jimentwood** [comment]
I'd suggest adding informative error message now.
And that we keep this in mind to adjust based on user feedback.

Some >50MB submission uploads for 'anc' content could useful, but I would suggest we wait until there is a clear need for it.

**Comment by erickpeirson** [comment]
We will need to extend the [default backend](https://github.com/arXiv/arxiv-base/tree/master/fourohfour) to implement the custom error backend behavior described [here](https://kubernetes.github.io/ingress-nginx/examples/customization/custom-errors/). 

Here is a nice example in Go: https://github.com/kubernetes/ingress-nginx/blob/master/images/custom-error-pages/main.go
